Pea75x/spanish-project-ruby
# Implement-sidekiq

### Changes 
- Created a background job to send out a weekly email to each high scorer for every game
- The emails should go out every Friday at midday
- The email should include the name of the game and their total score overall

**To test out NotificationJob -** 
Start redis server - 
`brew services start redis`
Verify redis is running - 
`redis-cli ping` you should receive PONG back if its working
Run sidekiq - 
`bundle exec sidekiq`
You should see this in the terminal - 
![image](https://github.com/user-attachments/assets/8076c7d3-00b9-4445-9627-d967a15d586d)
In another terminal run the server - 
`rails s`

You can test the _schedule.yml_ by changing the cron from every friday at midday

You should receive a pop up - 
![image](https://github.com/user-attachments/assets/6a9d2e9c-91c2-4bd1-a0d4-88adaaac9e4c)
